dls-controls/malcolmjs
# Feature/child panel visible #274

## Description

Added all the various interactions for when the child panel open state should be changed

## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- Start the site and run through the conditions in the story, (alternatively run the e2e tests to see all the cases get checked)

## Agile board tracking

connect to #274 
closes #274
